.github/workflows/continous-integration.yml

```diff
@@ -311,6 +311,20 @@ jobs:
         COVERALLS_PARALLEL: true
       run: poetry run coveralls
 
+  complete_coverage:
+    name: Complete coverage
+    runs-on: ubuntu-20.04
+    needs: [ test, changes ]
+    if: needs.changes.outputs.backend == 'true'
+
+    steps:
+      - name: Tell coveralls we're finished reporting
+        uses: coverallsapp/github-action@master
```

Comment:
should we pin a specific version / hash to avoid that they potentially break our CI with changes?

Reply:
We had `master` on `rasa-x` for ages without problems :thinking: plus, their past release was september last year -- coveralls seems to not be that well maintained tbh